test/framework/integration_framework/fake_peer/types.hpp

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef INTEGRATION_FRAMEWORK_FAKE_PEER_TYPES_HPP_
+#define INTEGRATION_FRAMEWORK_FAKE_PEER_TYPES_HPP_
+
+#include <functional>
+#include <memory>
+#include <vector>
+
+#include <boost/optional.hpp>
+#include "interfaces/common_objects/types.hpp"
```

Comment:
`interfaces/common_objects/types.hpp` should be included in cpp files reviewed above.

Reply:
The file `fake_peer/types.hpp` consists of type aliases and forward declarations, whereas the cpp files include the appropriate implementation headers.